### PR TITLE
Release v0.1.1, Pull image from MCR and documentation update

### PR DESCRIPTION
**Pull request by vinli-cn**
**What type of PR is this?**
/kind api-change
/kind documentation

**What this PR does / why we need it**:
Release v0.1.1, Pull image from MCR and documentation update
**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

File: README.md [hunk 1]
```diff
@@ -6,38 +6,54 @@
...
 
 ### Container Images & Kubernetes Compatibility:
 
-|driver version  |Image                                             | supported k8s version |
-|----------------|--------------------------------------------------|-----------------------|
-|main branch     |mcr.microsoft.com/k8s/csi/azurelustre-csi:latest  | 1.21+                 |
+| Driver version  | Image                                                         | Supported k8s version | Lustre client version |
+|-----------------|---------------------------------------------------------------|-----------------------|-----------------------|
+| main branch     | mcr.microsoft.com/oss/kubernetes-csi/azurelustre-csi:latest   | 1.21+                 | 2.14                  |
+| v0.1.1          | mcr.microsoft.com/oss/kubernetes-csi/azurelustre-csi:v.0.1.1  | 1.21+                 | 2.14                  |
```
Comment (andyzhangx) [inline README.md:21]: typo: `v.0.1.1`

Comment (vinli-cn) [reply README.md:21]: Thanks Andy! I will fix it in 0.1.3 release

**Pull request merged**